src/Templates/src/templates/maui-blazor-solution/.template.config/template.json

```diff
@@ -15,11 +15,9 @@
   "sourceName": "MauiApp.1",
   "primaryOutputs": [
     {
-      "condition": "(HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
       "path": "MauiApp.1.Shared/Pages/Home.razor"
     },
     {
-      "condition": "(HostIdentifier == \"dotnetcli\" || HostIdentifier == \"dotnetcli-preview\")",
       "path": "MauiApp.1.sln"
     },
```

Comment:
This is just telling VS that it should open these files. If you do this, the CLI will now have an error saying it can't open files.

Reply:
Totally fair here. I am still relatively new to templates but trying to get this one to work for non-VS users.